alfalfa_worker/model_advancer_base.py

```diff
@@ -27,14 +27,17 @@
 import datetime
 
 # sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
-from alfalfa_worker.lib.alfalfa_connections import AlfalfaConnections
-from alfalfa_worker.model_logger import ModelLogger
+from alfalfa_worker.lib.alfalfa_connections_base import AlfalfaConnectionsBase
+from alfalfa_worker.lib.logger_mixins import ModelLoggerMixin
 
 
-class ModelAdvancer(object):
-    """Base class for advancing models"""
+class ModelAdvancerBase(ModelLoggerMixin, AlfalfaConnectionsBase):
```

Comment:
Yes this looks like multiple inheritance and it kinda is. Mixins will call the remaining `__init__` methods.